image-container/stories/index.stories.js

```diff
@@ -0,0 +1,17 @@
+import '~/image-container/dockit-image-container.define';
+import '~/caption/dockit-caption.define';
+
+export default {
+  parameters: {
+    layout: 'centered',
+  },
+};
+
+export const image_container = () => /*html*/ `
+  <div>
+    <dockit-image-container>
+      <img src="https://picsum.photos/300" alt="Random image"></img>
+    </dockit-image-container>
+    <dockit-caption>The caption is here</dockit-caption>
+  </div>
```

Comment:
IIRC we had a call on that and the general idea was the we can just borrow the figure/figcaption naming and API as much as possible https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption

e.g. it can be `dockit-figure` instead of `dockit-image-container`?

Reply:
It's a good point, we should even remove `dockit-image-container` and just provide CSS for `figure` and `figcaption` in our default layouts.

This way users use standard HTML, it's better for a11y.

We don't need web components for this.